core/server/proxy/src/main/java/alluxio/proxy/s3/TaggingData.java

```diff
@@ -23,18 +28,28 @@
 import java.io.ObjectOutputStream;
 import java.io.Serializable;
 import java.util.ArrayList;
+import java.util.HashMap;
 import java.util.HashSet;
 import java.util.List;
+import java.util.Map;
 import java.util.Set;
+import javax.annotation.Nullable;
+import javax.xml.bind.annotation.XmlTransient;
 
 /**
  * Parsed version of the S3 tagging XML.
  * Used by the following S3 API actions:
  * - GetBucketTagging, GetObjectTagging, PutBucketTagging, PutObjectTagging
  */
 @JacksonXmlRootElement(localName = "Tagging")
+@JsonIgnoreProperties(ignoreUnknown = true, value = {"tagMap"})
```

Comment:
I was unable to marshal this struct while ignoring the metadata field `mTagMap` without specifying the rendered name `tagMap` in this annotation.

I'm not sure why `@XmlTransient` was insufficient, nor do I know if I have redundant/no-op annotations at this point.